### DISCUSSION TITLE
Enable kubernetes mode with Docker-CE in linux

### DISCUSSION THREAD

**Discussion by nauar** [BretFisher/ama#177] [Q&A]
Hello Bret,

I know how to do it in MAC and Windows with the Docker-EE version, but in linux platforms how can we enable enable kubernetes mode with Docker-CE? If this can't be done, then kubernetes-mode is only available in only a few payment distros, such as RedHat, where Docker-EE can be installed, isn't it?

Thank you very much!

Arnau Oncins

**Comment by BretFisher** [comment]
Surprise! This was just released at DockerCon as Docker open-sourced the Kubernetes "controller" for Compose. https://github.com/docker/compose-on-kubernetes
